R/FLORAL.R

```diff
@@ -265,24 +293,35 @@ FLORAL <- function(x,
     
     res$step2.tables <- list(min=character(0),
                              `1se`=character(0))
-
+    
     if (length(res$selected.feature$min.2stage)>0){
       namemat <- matrix(names(res$best.beta$min)[res$step2.feature.min],nrow=2)
       res$step2.ratios$min <- as.vector(na.omit(apply(namemat,2,function(x) ifelse(sum(is.na(x))==0,paste(x,collapse ="/"),NA))))
-      res$step2.ratios$min.idx <- res$step2.feature.min[,!is.na(colSums(res$step2.feature.min))]
+      res$step2.ratios$min.idx <- res$step2.feature.min#[,!is.na(colSums(res$step2.feature.min))]
```

Comment:
when I see commented out code like this, I wonder if it makes sense for any of it to be converted to a ticket instead.